UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -1,7 +1,8 @@
-﻿using Microsoft.CodeAnalysis;
+﻿#define WARNING_POPUP
```

Comment:
?

Reply:
I accidentally committed some of the code I wrote to help debug